_2021-finalproject/README.md

```diff
@@ -21,7 +21,7 @@ PR 考察分两阶段
 2. 第二阶段要求在11月10日21:00前完成合并
    - 值得注意的是，在第二阶段仍然可以修改或增加你的作业内容，但建议你尽量在这之前完成
    - 我们会在两个阶段的临界点修改其中一个检查项目，你需要在这之后再次通过所有检查项目，通过后再次 request review，如果没有问题，你的 PR 就会被合并
-   - 被修改的检查项目为：作业提交文件夹变更为 `_2021-finalproject`
+

```

Comment:
Don't change this file in your commit. After fix this, your PR are able to merge

Reply:
> Don't change this file in your commit. After fix this, your PR are able to merge

Is it  right now？

Reply:
Yes

Reply:
> Yes

Ok，thanks